tests/specs/packages.js

```diff
@@ -0,0 +1,190 @@
+define(["dist/amd-loader"], function(AMDLoader) {
+  describe("Test suite", function() {
+    var bitloader;
+    beforeEach(function() {
```

Comment:
I wouldn't do a `before` in this for example, because each time there is a call to `import` the cache gets populated.  So it is very important to keep things fresh for each unit test.  However, in the test below where I commented about using `before` is different because we are testing the output of the same action (same test).  So running `import` twice to test the same output is not necessary.